src/ArduinoNvs.cpp

```diff
@@ -166,7 +166,7 @@ bool ArduinoNvs::setBlob(String key, std::vector<uint8_t> &blob, bool forceCommi
   return setBlob(key, &blob[0], blob.size(), forceCommit);
 }
 
-int64_t ArduinoNvs::getInt(String key)
+int64_t ArduinoNvs::getInt(String key, int32_t default_value)
```

Comment:
Hi!
this function could return `int64_t`, so a `def_value` should be at least `int64_t`

Reply:
ok, thanks for correction